install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -84,11 +84,21 @@ spec:
 {{- end}}
 {{- if .Values.enableAnalysis }}
           - --enableAnalysis=true
+{{- end }}
+{{- if .Values.global.certificates }}
+          - --validation.tls.clientCertificate=/etc/dnscerts/cert-chain.pem
+          - --validation.tls.privateKey=/etc/dnscerts/key.pem
+          - --validation.tls.caCertificates=/etc/dnscerts/root-cert.pem
```

Comment:
This presumes the list always includes the galley DNS cert. Maybe check if `dns.istio-galley-service-account` is in the list?

Reply:
It is not straightforward for helm to parse .Values.global.certificates, understanding its proto structure and doing the check. Thus, in current implementation, similar to the approach for configuring other helm values, the user configuring .Values.global.certificates needs to ensure the value is valid.